vendor.go

```diff
@@ -131,16 +131,12 @@ func vendorPackages(depsGoHome string, projectdir string) {
 		if info == nil {
 			return nil
 		}
-		if info.IsDir() {
-			return nil
-		}
-		// Not sure this works, but seems feasible to ignore
-		// vendoring from libs, they should not do that
-		if strings.Contains(path, "/vendor/") {
-			return nil
+
+		if ignore, err := ignorePath(path, info.IsDir()); ignore {
+			return err
 		}
 
-		if !strings.HasSuffix(path, ".go") {
+		if info.IsDir() {
```

Comment:
Hmm cant this check also be part of the ignorePath ? (return true, nil) ?

Reply:
I kept then separate because `ignorePath` ignores (by skipping) the directories we don't want to recurse. But in this line, we're telling the walker to continue processing files in this directory (in other words, it's legitimate directory). 
Maybe if we rename the ignore function to something else we can do all-in-one function.

Reply:
Well, the current function, when it is a file, also tells the walker to ignore a path + continue processing files on the dir, in case the file should be ignored but not the whole dir. So it seems that the function already does "ignore path but won't skip dir" logic. It was a little confusing to me why it doesn't do all skip/ignore logic in one place. More because the function name already doesn't mention file or dir, it is path, so it already conveys the idea it will work on both, and it does return two value, if it must ignore and if it must skip the current dir, so it feels like it should solve the whole ignoring/skipping problem.

Reply:
You're right. I will change.